TACC/Core-CMS-Custom
# Task/FP-1908 - Set default value for registration_status

## Overview

Minor changes to the Registration Form

## Related

- [FP-1908](https://jira.tacc.utexas.edu/browse/FP-1908)

## Changes

- Set a default value for Registration Status to "Received"
- Prevent form caching on reload/back navigation (prevent the same form data from being reused)
- Include a missing parenthesis for Entity Being Registered

## Testing


1. Fill out some fields on the [form](http://localhost:8000/register/request-to-submit/) (no need to submit)
2. Navigate to another page, and then use the back button to try returning to the form
3. The form should be cleared
4. No means to test database submissions locally at the moment
